lib/grape/validations/validators/default.rb

```diff
@@ -6,8 +6,30 @@ def initialize(attrs, options, required, scope)
         super
       end
 
+      # return true if we know we cannot dup this object
+      private def cannot_dup(obj)
```

Comment:
Should be `....?`, but I would write this as the other way around, so maybe `duplicatable?`, which is how ActiveRecord calls these things I believe.